examples/post_training_quantization/openvino/yolov8_quantize_with_accuracy_control/main.py

```diff
@@ -165,13 +167,11 @@ def validation_ac(
                 ]
             preds = validator.postprocess(preds)
             validator.update_metrics(preds, batch)
-            counter += 1
         stats = validator.get_stats()
         if num_outputs == 1:
             stats_metrics = stats["metrics/mAP50-95(B)"]
         else:
             stats_metrics = stats["metrics/mAP50-95(M)"]
-        print(f"Validate: dataset length = {counter}, metric value = {stats_metrics:.3f}")
```

Comment:
Could you provide the output after your changes with timestamp? I would like to make sure that the interactivity of the example has not degenerated.

Reply:
<img width="960" alt="image" src="https://github.com/user-attachments/assets/1f58b4b8-e1c9-452b-9d26-c554b0fdf587">
After add track in collect_values_for_each_item_using_prepared_model